person.rb

```diff
@@ -1,22 +1,28 @@
 require_relative 'nameable'
 require './decorator'
+require_relative 'rental'
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :rental, :parent_permission
```

Comment:
- [ ] I believe this should be `:rentals` as we have in the constructor method. Kindly rename it.